tests/fixtures/tests.py

```diff
@@ -917,14 +917,12 @@ def test_loaddata_error_message(self):
             management.call_command("loaddata", "invalid.json", verbosity=0)
 
     @unittest.skipUnless(
-        connection.vendor == "postgresql", "psycopg2 prohibits null characters in data."
+        connection.vendor == "postgresql",
+        "PostgreSQL prohibits null characters in data.",
```

Comment:
This can be reverted.

Reply:
Why?

Reply:
Because it's just an unrelated indentation change.

Reply:
Ah, understood :)

Reply:
Sadly enough the change is unrelated but `PostgreSQL` is longer than `psycopg2` and as such black rewrites it to this and pre-commit won't let me commit it without breaking in multiple lines. I think we have to leave it as is?

Reply:
True, sorry, I didn't notice that it was `psycopg2`.